EIPS/eip-1538.md

```diff
@@ -3,12 +3,13 @@ eip: 1538
 title: Transparent Contract Standard
 author: Nick Mudge <nick@mokens.io>
 discussions-to: https://github.com/ethereum/EIPs/issues/1538
-status: Draft
+status: Abandoned
```

Comment:
Can we just delete this EIP instead?  This repository has a *huge* volume of DRAFT EIPs that are not adding value by sitting idle.  If `nick@mokens.io` agrees, I would prefer to delete rather than mark as abandoned.  We have git history, so its not like the work will be lost forever.

Reply:
No because EIP-1538 is in use by contracts in production and is being linked to by various places on the Internet. In addition the final EIP-1155 suggests using EIP-1538 for upgrading contracts.

Reply:
In addition EIP-1538 is part of the development and history of EIP-2535.

Reply:
> Editor Feedback:
> 
> * address the "who is Nick Mudge" question I had (different email addresses)
> * make section headings match template section headings
> 
> Personal Feedback:
> 
> * I'm not a fan of the themed naming convention.  I think standards should name things in a way that makes them immediately understandable to the widest audience possible while conveying what they do in a pithy manner.  Following a naming theme in the contract like `loupe`, `facets` and `cut` is cute but not useful for understanding.

I made the fixes.  Thanks very much for your feedback and help this.

You are right about the terminology and it has been debated.  Terminology has been in use for 6 months and we are stuck with it now.